include/digits.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef OBC_DIGITS_HPP
+#define OBC_DIGITS_HPP
+
+#include <Arduino.h>
+
+namespace obc {
+
+template <typename T>  // requires std::unsigned_integral<T>
+constexpr bool has_tens_digit(T n)
+{
+    return n > 9;
+}
+
+template <typename T>  // requires std::unsigned_integral<T>
+constexpr bool has_hundreds_digit(T n)
+{
+    return n > 99;
+}
+
+}  // namespace obc
+
+#endif
```

Comment:
używane tylko w serializacji, ja bym to przeniósł do logger.cpp jako funkcja

Reply:
z tego co paietam to jest uzywane jeszcze w printach w gps.cpp. Ale tam duplikacja kodu jest koszmarna i trzeba cos z tym zrobic

Reply:
generalnie używane tylko na obiektach Gps<cośtam> wiec może przenieść do gps.hpp?